config/nrfconnect/.gn

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The location of the build configuration file.
+buildconfig = "//build/config/BUILDCONFIG.gn"
+
+# CHIP uses angle bracket includes.
+check_system_includes = true
+
+# Allow loading paths relative to //gn.
+secondary_source = "//third_party/connectedhomeip/gn/"
+
+default_args = {
+  # We don't want the build scripts to inject any additional CPU related compiler flags,
+  # because those are supplied from cmake.
+  target_cpu = "zephyr"
```

Comment:
We should specify the actual CPU.

There is no problem adding a condition
if (current_os != "zephyr")

where the flags are being added. Hiding the CPU isn't a good solution.

Reply:
Done. I have tried just setting `arm` again and checked the output. It looks like cpu flags are not duplicated, so it's okay in my book.